app/src/main/java/com/example/demo_mvp_zomatofood/data/source/remote/fetchjson/GetJsonFromUrl.kt

```diff
@@ -0,0 +1,36 @@
+@file:Suppress("DEPRECATION")
+
+package com.example.demo_mvp_zomatofood.data.source.remote.fetchjson
+
+import android.os.AsyncTask
+import com.example.demo_mvp_zomatofood.data.source.remote.OnFetchDataListener
+import org.json.JSONObject
+
+@Suppress("DEPRECATION")
+class GetJsonFromUrl<T>(
+    private val listener: OnFetchDataListener<T>,
+    private val keyEntity: String
+) : AsyncTask<String?, Void?, String?>() {
```

Comment:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#29-parameter-ordering-in-methods